web/portal/platform/src/router/EntityMap.ts

```diff
@@ -54,6 +54,14 @@ const getEntityMap = (): ExtendedRouteMap => {
         },
       ],
     },
+    {
+      label: _('Calls'),
```

Comment:
Active calls require a custom component and route. Check this out https://github.com/irontec/ivozprovider/blob/bleeding/web/portal/brand/src/router/addCustomRoutes.ts